src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -6532,14 +6539,21 @@ void CodeGen::genJmpMethod(GenTree* jmp)
             regSet.AddMaskVars(genRegMask(argReg));
             gcInfo.gcMarkRegPtrVal(argReg, loadType);
 
-            if (compiler->lvaIsMultiregStruct(varDsc, compiler->info.compIsVarArgs))
+            if (varDsc->GetOtherArgReg() < REG_STK)
```

Comment:
Because ARM64 and RISCV have different calling convention. Right, ARM64 seems more clearer, but RISCV follows LA64's. So we need to update like LA64.
`GetOtherArgReg` is the second register for multi argument register for RISCV and LA64.

In case of ARM64 in the test, it passes two float register arguments using one int register (REG_A0). So `compiler->lvaIsMultiregStruct` returns `false`. In method prolog in a callee, it makes two float values from REG_A0 in ARM64.